src/Assets/LightSkyDefense/Scripts/GameManager.cs

```diff
@@ -4,16 +4,84 @@
 
 public class GameManager : MonoBehaviour
 {
-    // Start is called before the first frame update
-    void Start()
+    [HideInInspector]
+    public GameObject _wayPointPrefab;
```

Comment:
I think we could just set this value in scene, setting the value of the `_wayPointPrefab` using a script feels like a hack to me.